repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/searchresults-main.ftl

```diff
@@ -39,24 +39,24 @@
             </form>
             <div class="nhsuk-listing">
```

Comment:
I think we may need to comment or refactor `uk.nhs.hee.web.components.SearchResultsComponent` class to not to use `contentTypesMap` & `selectedContentTypes` and it's references.

Reply:
Done. Commented for now.